ComfyUI/Dockerfile.prod

```
@@ -0,0 +1,83 @@
+  for path in "\$@"; do
+    target="${COMFYUI_PATH}/\${path#./}"
+    mkdir -p "\$(dirname "\$target")"
+    mv -f -- "\$path" "\$target"
+  done
+' _ {} +
+fi
+cd ${COMFYUI_PATH}
+comfy node fix all
+comfy launch -- --listen * --port 8188 --disable-auto-launch --gpu
+EOF
+RUN cat ${ENTRYPOINT_SCRIPT_NAME}
```

Hadolint (2.12.0)

[error] unexpected 'R'
expecting a new line followed by the next instruction

(DL1000)